svcdef/consolidate_http.go

```diff
@@ -22,13 +25,13 @@ func consolidateHTTP(sd *Svcdef, protoFiles []io.Reader) error {
 		lex := svcparse.NewSvcLexer(pfile)
 		protosvc, err := svcparse.ParseService(lex)
 		if err != nil {
-			if strings.Contains("'options' or", err.Error()) {
+			if strings.Contains(err.Error(), "'option' or 'additional_bindings'") {
```

Comment:
Because we are using `pkg/errors` I believe we want an `err.Cause().Error()` here. 

Also how does the "'this' or 'that'" syntax work for `strings.Contains`?